notebooks/text_classification/solutions/2_keras_for_text_classification.ipynb

```diff
@@ -910,6 +910,218 @@
     "cnn_model.summary()"
```

Comment:
<p>UNITS hasn't been defined yet at this point.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='89'/>

Reply:
<p>I think it has on line 11 in the same cell, hasn't it?</p>